src/server/mod.rs

```diff
@@ -51,7 +53,15 @@ impl<P: Publisher> Server<P> {
             anyhow::bail!("Given prefix was empty")
         }
 
-        let topic = format!("{prefix}.>");
+        let topic_prefix = if multitenant {
+            format!("*.{prefix}")
+        } else {
+            prefix.clone()
+        };
+
+        let topic = format!("{topic_prefix}.>");
+        eprintln!("GONNA LISTEN ON: {topic}");
```

Comment:
Oopsie whoopsie

Reply:
I had full confidence and definitely never needed to verify this